apps/generic_browser.talon

```diff
@@ -1,6 +1,7 @@
 tag: browser
 -
 (address bar | go address | go url): browser.focus_address()
+(address copy | url copy | copy address | copy url ): user.address_copy()
```

Comment:
Quite a few different commands here as I try to stay consistent with the existing commands in the file but also meet the desired object -> verb standards